ichk/command.py

```diff
@@ -32,6 +32,8 @@ def entry():
     parser.add_argument("-T", "--timeout", default=10*60, type=int,
                         help="Sets the maximum amount of seconds to wait for server responses"
                             +", default 600. Increase this to account for longer-running queries.")
+    parser.add_argument("-s", "--set-root-collection", dest='root_collection', default=None,
```

Comment:
`--root-collection` would be more consistent.
```suggestion
    parser.add_argument("-s", "--root-collection", dest='root_collection', default=None,
```